### DISCUSSION TITLE
Unable to retrieve history. See details in developer tools console.

### DISCUSSION THREAD

**Discussion by krishnamurthi-cpu** [StackStorm/st2#5746] [Polls]
Hi,

Getting this error "Unable to retrieve history. See details in developer tools console." while clicking in History tab in stackstorm UI. How can I resolve this issue.

**Comment by amanda11** [comment]
When errors are found executing an action in the Web UI then they are written to the brower's console. This should give you some indication of the error.

What does the Developer Tools Console show?

For example, on Chrome you get to it via the menu on the right hand side, and then More tools -> Developer tools. Then when you get the panel with the developer, choose the "Console" tab to see what error is returned. If this doesn't give a useful error, check also what the Network tab shows.
